### PR TITLE
Make the atmos firefighter suit waterproof

### PR DESCRIPTION
**Pull request by datlo**
This PR changes the Atmospherics Firefighter suit to be waterproof.

The emergency firesuit and atmos hardsuit aren't affected. This gives an upside to the firefighter suit at the cost of spaceproofing.

Reason : Fighting fires with something that kills you if you make the slightest mistake is frustrating, to say the least. :alien:

🆑
Add: Atmospherics firefighter suit is now waterproof, atmos hardsuit is not changed.
/🆑

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -497,7 +497,14 @@ emp_act
 			return 0
 	..()
 
+/mob/living/carbon/human/fire_act()
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human_defense.dm]: So people can no longer turn themselves into blazing infernos and be immune to it?

Yeah, I see no reason to remove this at all. It's fun and *cool*.

**Comment by Fox-McCloud** [comment]
Making entire systematic changes for an inconvenience that's intentionally designed and placed in for a singular race is....not good design.

**Comment by datlo** [comment]
Guess it's just a disagreement of opinion, then.

I'll remove the fireproofing part of this PR in a moment.

**Comment by datlo** [comment]
I also removed waterproofing on the CE/atmos hardsuits to make the choice between atmos hardsuit and firefighter suit more interesting.

**Comment by Allfd** [comment]
Sorry, the Suit that is immune to all possible ranges of pressures and temperatures is now no longer water proof?

**Comment by datlo** [comment]
If we went that way, all hardsuits should be waterproof. But then that's a balance concern or something.

Eeh. I need to think.

**Pull request closed without merging**